From b20fc54c08e8afe1365e56cacade3ec39984da8d Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 18 Apr 2017 19:00:34 -0400
Subject: [PATCH 04/29] Fix "certficate" argument name.

This fixes our typoed argument name by making the incorrectly spelled
version be a popt alias, and fixing the real implementation to be
spelled right in pesign.c .

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/pesign.c    | 2 +-
 src/pesign.popt | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/pesign.c b/src/pesign.c
index af374b6..279a17a 100644
--- a/src/pesign.c
+++ b/src/pesign.c
@@ -438,7 +438,7 @@ main(int argc, char *argv[])
 		 .arg = &ctxp->outfile,
 		 .descrip = "specify output file",
 		 .argDescrip = "<outfile>" },
-		{.longName = "certficate",
+		{.longName = "certificate",
 		 .shortName = 'c',
 		 .argInfo = POPT_ARG_STRING,
 		 .arg = &certname,
diff --git a/src/pesign.popt b/src/pesign.popt
index 7b3385d..5a97748 100644
--- a/src/pesign.popt
+++ b/src/pesign.popt
@@ -1,2 +1,3 @@
 pesign alias --cert --certificate
+pesign alias --certficate --certificate
 pesign alias --daemon --daemonize
-- 
2.13.4

